charters/ingestion-service/charter.md

```diff
@@ -0,0 +1,177 @@
+
+### Supporting tools
+
+* Generation of lab specific spreadsheets that are not directly generated from a JSON schema endorsed by the metadata team. Additional customisations (for example, modifying column names or adding additional tabs) for individual contributors will not be supported through any mechanism other than through the use of standard configuration (i.e. which modules are to be included) or through the proposal of metadata schema changes
+
+## Milestones and Deliverables
+
+  1. _**"Wrangler-assisted"** submission phase:_
+     - Provision of services to allow data contributors to upload all required data and metadata. Process is heavily wrangler-led, with metadata exchanged manually (e.g. via spreadsheets emails to wranglers) and support provided to upload data. Contributors see very few DCP user interfaces
+     - Delivery: **Q3 2018**
+  1. _**"Online, wrangler assisted"** submission phase:_
+     - Provision of services to allow wrangler-assisted submissions with user interfaces available to contributors to provide additional support, including views over all submissions for a single project, summaries of submissions in progress, and validation reports for submissions
+     - Delivery: **Q1 2019**
+  1. _**"Self-service"** submission phase:_
+     - Delivery of services to transition from a wrangler-led submission process to a "self-service" submission process, including full-featured user interfaces to support spreadsheet based submissions
+     - Delivery: **Q3 2019**
+  1. _**"Self-service, review and repair"** submission phase:_
+     - Delivery of services to provide support for the review and repair of potential errors or inconsistencies in submissions, including reviewing validation errors and providing updates for instant revalidation, reviewing ontology mappings, providing additional ontology mappings in a responsive user interface.
+     - Delivery: **Q4 2019**
+  1. _**"Dynamic evolution"** submission phase:_
+     - Provision of ingest services that support data contributors in adding new fields to their submissions, which dynamically triggers metadata updates for the metadata team to review and accept during the submission process. Submitters are informed, via summary views in the user interface, when their data has been used in an analysis pipeline and results are available. Progress of submissions throughout the system - including references to external usage (e.g. via DCP data portals) can be viewed supporting transparent data attribution
+     - Delivery: **2020**
+
+## Roles
+
+### Project Lead
+
+[Tony Burdett](mailto:tburdett@ebi.ac.uk)
+
+### Product Owner
+
+[Norman Morrison](mailto:norman@ebi.ac.uk)
+
+### Technical Lead
+
+[Simon Jupp](mailto:jupp@ebi.ac.uk) (Interim, pending new hire)
```

Comment:
Can we add the new hire here yet?

Reply:
I'll update the charter once it is official